spec/features/projects_spec.rb

```diff
@@ -157,10 +157,101 @@
       sleep FeatureHelpers::TIME_TO_SLEEP
       fill_in "project_posts_attributes_0_title", with: 'Foo title'
       page.execute_script( "$('.redactor').redactor('code.set', 'Bar')" )
-      
       click_button(I18n.t('projects.dashboard_posts.publish'))
       expect(page).to have_content('Foo title')
     end
   end
 
+  describe "contribute" do
+    let(:project){ create(:project, state: 'online') }
+    let(:successful_project){ create(:project, state: 'successful') }
+    let(:failed_project){ create(:project, state: 'failed') }
+
+    before do  
```

Comment:
I think we should move to the before block the lines
```ruby
5.times{ create(:reward, project: project) }
visit project_path(project)
```
Int the case where you have the ```visit project_path(successful_project)``` we can solve this with a ```let(:project){ successful_project }```inside a context.